pkg/sentry/server/server.go

```diff
@@ -184,6 +184,8 @@ func (s *Server) signCertificate(ctx context.Context, req *sentryv1pb.SignCertif
 		dns = []string{fmt.Sprintf("dapr-sidecar-injector.%s.svc", req.GetNamespace())}
 	case req.GetNamespace() == security.CurrentNamespace() && req.GetId() == "dapr-operator":
 		dns = []string{fmt.Sprintf("dapr-webhook.%s.svc", req.GetNamespace())}
+	case req.GetNamespace() == security.CurrentNamespace() && req.GetId() == "dapr-scheduler":
+		dns = []string{fmt.Sprintf("dapr-scheduler-server.%s.svc", req.GetNamespace())}
```

Comment:
Should we make this `dapr-scheduler-server-a` to match what we did in [this pr](https://github.com/dapr/dapr/pull/8401/files#diff-4d6aae0e9704c4ff709a3ff0afaa2d61e740bf745d2fafdece7bc859381aac40)?

Reply:
The ETCD DNS lookup fails with `-a` so we need to use this service. This means we will always need both Services now- this is fine.